tests/sentry/api/endpoints/test_project_rule_details.py

```diff
@@ -160,6 +167,30 @@ def test_with_filters(self):
         assert len(response.data["filters"]) == 1
         assert response.data["filters"][0]["id"] == conditions[1]["id"]
 
+    @responses.activate
+    def test_neglected_rule(self):
+        now = datetime.now().replace(tzinfo=timezone.utc)
+        NeglectedRule.objects.create(
+            rule=self.rule,
+            organization=self.organization,
+            opted_out=False,
+            disable_date=now + timedelta(days=14),
+            sent_initial_email_date=now,
```

Comment:
Won't need these 2 lines once I merge a migration to fix this - these should be nullable.

Reply:
Can remove once https://github.com/getsentry/sentry/pull/56309 is merged